src/simsopt/field/coil.py

```diff
@@ -186,6 +186,39 @@ def coils_via_symmetries(curves, currents, nfp, stellsym):
     return coils
 
 
+def load_coils_from_file(filename, order, ppp=20, maxiter=1000, tol=1e-8):
```

Comment:
Can you use `load_coils_from_makegrid_file`?  We may have different file formats and don't want to confuse users.